README.md

```diff
@@ -4,6 +4,21 @@
 
 The aws-operator manages Kubernetes clusters running on AWS.
 
+## Branches
+
+- `thiccc`
+    - Up to version v5.4.0.
```

Comment:
Up to an including, or up to and excluding?

Reply:
And same further, there is the possibility for ambiguity here.

Reply:
Everything is inclusive here.

Reply:
Can we be explicit about that?

Reply:
Suggestions please <3